packages/hooks/CHANGELOG.md

```diff
@@ -10,6 +10,7 @@
 
 #### Changed
 
+- Removed parsePrice from package. [#106](https://github.com/liteflow-labs/liteflow-js/pull/106)
```

Comment:
This is a breaking change. Can you move it to the breaking change section?

Reply:
done